File: Adriano/Cliente.cs
public class Cliente {
    public int Id { get; set; }
    public string Nome { get; set; }
    public string Endereco { get; set; }
    public string Cpf { get; set; }

    public Cliente () {}

    public Cliente (int id, string nome, string endereco, string cpf) {
        Id = id;
        Nome = nome;
        Endereco = endereco;
        Cpf = cpf;
    }
}

File: Tiago_sem_h/Cliente.java
public class Cliente{
    private String nome;
    private int idade;
    private String endereco;

    public Cliente(String nome,int idade,String endereco){
        this.nome = nome;
        this.idade = idade;
        this.endereco = endereco;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public String getEndereco() {
        return endereco;
    }

    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }

    @Override
    public String toString() {
        return "Cliente [endereco=" + endereco + ", idade=" + idade + ", nome=" + nome + "]";
    }
}